src/__tests__/utils/event-utils.test.ts

```diff
@@ -160,15 +182,6 @@ describe(`event-utils`, () => {
                 expectedVersion: 10.1,
                 expectedBrowser: 'BlackBerry',
             },
-            {
-                name: 'Android Mobile',
-                userAgent:
-                    'Mozilla/5.0 (Linux; StarOS Must use __system_property_read_callback() to read; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/100.0.4896.127 Mobile Safari/537.36',
```

Comment:
I deleted this test as it looks like a typo, surely this isn't a real user agent?

Reply:
the test cases are all copied in from elsewhere... but even if it's real I'll guess it's rare enough we don't care 😊